_studio/mfx_lib/decode/vp9/src/mfx_vp9_dec_decode_hw.cpp

```diff
@@ -945,6 +945,124 @@ mfxStatus VideoDECODEVP9_HW::PrepareInternalSurface(UMC::FrameMemID &mid, mfxFra
     return MFX_ERR_NONE;
 }
 
+
+static mfxStatus checkFrameInfoCommon(mfxFrameInfo  *info)
```

Comment:
U actually don't need to bring _checkFrameInfoCommon_ here, it's actually very common checks that are not dependent on codec.

Reply:
Yes, removed and refactored